app/src/main/java/com/globant/example/mentorapp/home/domain/model/UserEntity.java

```diff
@@ -0,0 +1,168 @@
+package com.globant.example.mentorapp.home.domain.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class UserEntity {
+
+    @Expose
+    private String login;
+    @Expose
+    private Integer id;
+    @SerializedName("avatar_url")
+    @Expose
+    private String avatarUrl;
+    @SerializedName("gravatar_id")
+    @Expose
+    private String gravatarId;
+    @Expose
+    private String url;
+    @SerializedName("html_url")
+    @Expose
+    private String htmlUrl;
+    @SerializedName("followers_url")
+    @Expose
+    private String followersUrl;
+    @SerializedName("following_url")
+    @Expose
+    private String followingUrl;
+    @SerializedName("gists_url")
+    @Expose
+    private String gistsUrl;
+    @SerializedName("starred_url")
+    @Expose
+    private String starredUrl;
+    @SerializedName("subscriptions_url")
+    @Expose
+    private String subscriptionsUrl;
+    @SerializedName("organizations_url")
+    @Expose
+    private String organizationsUrl;
+    @SerializedName("repos_url")
+    @Expose
+    private String reposUrl;
+    @SerializedName("events_url")
+    @Expose
+    private String eventsUrl;
+    @SerializedName("received_events_url")
+    @Expose
+    private String receivedEventsUrl;
+    @Expose
+    private String type;
+    @SerializedName("site_admin")
+    @Expose
+    private Boolean siteAdmin;
+
+    /**
+     *
+     * @param eventsUrl
+     * @param siteAdmin
+     * @param gistsUrl
+     * @param type
+     * @param gravatarId
+     * @param url
+     * @param subscriptionsUrl
+     * @param id
+     * @param followersUrl
+     * @param reposUrl
+     * @param htmlUrl
+     * @param receivedEventsUrl
+     * @param avatarUrl
+     * @param followingUrl
+     * @param login
+     * @param organizationsUrl
+     * @param starredUrl
+     */
+    public UserEntity(String login, Integer id, String avatarUrl, String gravatarId, String url, String htmlUrl,
```

Comment:
Ok, I thought Gson needed the constructor to create the objects. Constructor erased.